extensions/amp-script/0.1/test/integration/test-amp-script.js

```diff
@@ -78,5 +79,47 @@ describe.configure().skipSinglePass().run('amp-script', function() {
         return element.classList.contains('i-amphtml-broken');
       });
     });
+
+    it('should allow without gesture for small size-defined', function*() {
+      yield poll('<amp-script> to be hydrated',
+          () => element.classList.contains('i-amphtml-hydrated'));
+      const impl = element.implementation_;
+
+      // Give event listeners in hydration a moment to attach.
+      yield browser.wait(100);
+
+      sandbox.stub(impl.userActivation_, 'isActive').callsFake(() => false);
+      sandbox.stub(impl, 'getLayout').callsFake(() => Layout.FIXED);
+      sandbox.stub(impl, 'getLayoutBox').callsFake(() => {
+        return {height: 300};
+      });
```

Comment:
This `describes.integration` block uses layout=container in the element. Let's create another block with an fixed layout and size < 300 instead of stubbing.

Reply:
Done